service/title_formatter.py

```diff
@@ -98,3 +100,13 @@ def _format_private_individual_name(name):
     formatted_name = ' '.join(name_list)
     decoration = name.get('decoration')
     return '{0}, {1}'.format(formatted_name, decoration) if decoration else formatted_name
+
+# If the district field contains both the county and district, only display the district
+def _format_districts(districts):
+    formatted_districts = []
+    for district in districts:
+        if ':' in district:
+            formatted_districts.append(re.sub('.*: ', '', district));
+        else:
+            formatted_districts.append(district)
+    return formatted_districts;
```

Comment:
Think this could have been done in a few less lines. Don't need the if/else as the re.sub on line 109 will check for the ':' anyway.